.bazelignore

```diff
@@ -20,6 +20,10 @@ examples/bzlmod/other_module/bazel-out
 examples/bzlmod/other_module/bazel-testlogs
 examples/bzlmod/py_proto_library/foo_external
 examples/bzlmod_build_file_generation/bazel-bzlmod_build_file_generation
+examples/build_file_generation/bazel-bin
```

Comment:
These show up when doing a build in the root dir.

Should the entire `examples/` be ignored if those are all submodules?

Reply:
It all worked fine for me locally as well, but on CI bazel in the root dir was trying to compile things in `examples/build_file_generation/bazel-*` 🤷

Reply:
I can't find it atm, it should be in the CI results for https://github.com/bazel-contrib/rules_python/commit/7018ca442677a14e15ea055a879e2d2c00c11970 or a parent commit though

Reply:
Great! Though it does sound a bit like a CI flake...

Would you mind pushing an empty commit just to be sure? `git commit -m "empty" --allow-empty; git push`.

Reply:
You can see 1/2 failed...